modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -265,34 +367,25 @@ Returns five records starting from the second position.
 
 === Server-Side Pagination in APIkit for OData 4
 
-Your API implementation retrieves a limited number of records using a fix `pageSize` parameter. The response of the first request contains `@odata.nextlink` with a URL to get the next page.
+Server-side pagination is a way of controlling a subset of data requests that were fetched from a client-side server using a fixed `pageSize` parameter. It divides the response data set into discrete pages to improve data readability. The server defines the page size, which is the maximum number of records returned by the server in a request.
 
-For example:
+===== How server-side-pagination works in OData 4
 
-[source,curl]
---
-curl -X GET ‘localhost:8081/api/Orders’
---
+OData defines a `@odata.nextLink` to indicate that a response is only a subset of the requested collection of entities. As such, `@odata.nextLink` contains a URL that allows the client to retrieve the next subset of the requested collection.
+Apart from that, the URL contains a query parameter `$skiptoken` that indicates the server where the next page starts.
```

Comment:
Remove _Apart from that,_ and leave this sentence next to the previous one.

Reply:
Thanks!